site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+important that I'm going to write it one more time below for those who simply
+skim through that post.
+
+{% Aside 'gotchas' %}Don't assume a video will play, and don't show a pause
+button when the video is not actually playing.{% endAside %}
+
+You should always look at the [Promise](promise) returned by the play function
+to see if it was [rejected]:
+
+```js
+var promise = document.querySelector('video').play();
+
+if (promise !== undefined) {
+  promise.then(_ => {
+    // Autoplay started!
+  }).catch(error => {
+    // Autoplay was prevented.
+    // Show a "Play" button so that user can start playback.
+  });
+}
+```
+
+{% Aside 'caution' %}Don't play interstitial ads without showing any media
+controls as they may not autoplay and users will have no way of starting
+playback.{% endAside %}
+
+One cool way to engage users is to use muted autoplay and let them chose
+to unmute. (See the example below.) Some websites already do this effectively,
+including Facebook, Instagram, Twitter, and YouTube.
+
+```html
+<video id="video" muted autoplay>
+<button id="unmuteButton"></button>
+
+<script>
+  unmuteButton.addEventListener('click', function() {
+    video.muted = false;
+  });
+</script>
+```
+
+### Web Audio
+
+The [Web Audio API](web-audio) has been covered by autoplay since Chrome 71.
+There are a few things to know about it. First, it is good practice to wait for
+a user interaction before starting audio playback so that users is aware of
+something happening. Think of a "play" button or "on/off" switch for instance.
+You can also add an "unmute" button depending on the flow of the app.
+
+{% Aside %}If an `AudioContext` is created before the document receives a user
+gesture, it will be created in the "suspended" state, and you will need to call
+`resume()` after the user gesture.{% endAside %}
+
+If you create your `AudioContext` on page load, you'll have to call `resume()`
+at some time after the user interacted with the page (e.g., after a user clickes
+a button). Alternatively, the `AudioContext` will be resumed after a user
+gesture if `start()` is called on any attached node.
+
+```js
+// Existing code unchanged.
+window.onload = function() {
+  var context = new AudioContext();
+  // Setup all nodes
+  // ...
+}
+
+// One-liner to resume playback when user interacted with the page.
+document.querySelector('button').addEventListener('click', function() {
+  context.resume().then(() => {
+    console.log('Playback resumed successfully');
+  });
+});
+```
+
+You may also create the `AudioContext` only when the user interacts with the
+page.
+
+```js
+document.querySelector('button').addEventListener('click', function() {
+  var context = new AudioContext();
+  // Setup all nodes
+  // ...
+});
+```
+
+To detect whether the browser requires a user interaction to play audio, check
+`AudioContext.state` after you've created it. If your media is allowed to play,
```

Comment:
There's no "media". I believe it should "If you are allowed to play," as it's more generic. WebAudio is different than regular audio/video elements.

Reply:
Used "if playing is allowed."

Reply:
Fixed.